src/interfaces/alteration-interfaces/alteration-interface.ts

```diff
@@ -33,3 +33,9 @@ export interface AlterationIF extends CoopAlterationIF {
   contactPoint: ContactPointIF
   courtOrder?: CourtOrderIF
 }
+
+export interface EntityTypeOption {
+  value: string;
+  SHORT_DESC: string;
+  text: string;
+}
```

Comment:
As this isn't an "alteration interface", I recommend putting it in its own file, maybe.

Also, why "SHORT_DESC"? Could this follow our usual camel case convention?

Reply:
I moved to its own file, and changed to camel case. "SHORT_DESC" is from existing type alteration code.

Reply:
Great, thanks!